### ISSUE TITLE
update_check api is broken

### ISSUE THREAD

**Issue by tfonfara** [Tautulli/Tautulli#1264]
**Version:** v2.0.20-beta
**Branch:** beta
**Commit hash:** ab34a742107493e01ce0cbd75fce4e5bb3bef148
**Operating system:** Linux 4.4.0-116-generic
**Python version:** 2.7.14 (default, Dec 14 2017, 15:51:29) [GCC 6.4.0]
**What you did?** Called the new update_check api as shown in the docs
**What happened?** Error occured
**What you expected?** Api to work as expected
**How can we reproduce your issue?**

Seems that the update_check api is not working properly:
`https://<host>/api/v2?apikey=<apikey>&cmd=update_check`
Response:
`{"response": {"message": "Check the logs", "data": {}, "result": "error"}}`
Logs:
`Tautulli APIv2 :: API called with kwargs: {'cmd': u'update_check', 'apikey': u'********cd'}`
`Tautulli APIv2 :: Cleaned kwargs: {}`
`Tautulli APIv2 :: Failed to run update_check with {}: unsupported format character ')' (0x29) at index 16`

**Comment by TheMeanCanEHdian** [comment]
Works fine for me on the latest beta. If you still have issues come by the Discord. Github is for bug reports.

**Comment by tfonfara** [comment]
@Wreave if you test the "no updates" case everything works fine, did you also test the "there are updates available" case? If not please re-open the issue.

**Comment by TheMeanCanEHdian** [comment]
I did but now that I look I tested with a nightly update. So that is my mistake.

**Comment by tfonfara** [comment]
@JonnyWong16 thanks for fixing the issue! From your diff (02171882741092c37712658040117bc7f717afe5) I can see that the update check will return in two cases something like this:
```
{
	"response": {
		"message": "A new release of Tautulli is available",
		"update": true,
		"release": true,
		"latest_release": "...",
		"result": "success"
	}
}
```

Wouldn't it make more sense to keep it consistent to the other apis and return it in that structure?
```
{
	"response": {
		"message": "A new release of Tautulli is available",
		"data": {
			"update": true,
			"release": true,
			"latest_release": "..."
		},
		"result": "success"
	}
}
```

**Comment by JonnyWong16** [comment]
Fixed in v2.0.22-beta.
